### PR TITLE
Users models

### PR DESCRIPTION
**Pull request by aradz1997**
1) Created the models for the users app - User, Teams
2) Created the migration file for the app
3) Registered the models on admin.py
4) add 'nohup.out'  to .git ignore
5) installed Django on pip

### CODE REVIEW

**Review by orzionpour — APPROVED** [review]
LGTM
Resloves #25

**Review by Asafh7 — APPROVED** [review]

**Review by avraham1992 — CHANGES_REQUESTED** [review]
I need to see that it works with the task...marge them on your computer and upload them together because there is currently no connection between them

**Review by orzionpour — COMMENTED** [review]
After reading about django's auth model:
https://docs.djangoproject.com/en/3.2/ref/contrib/auth/

This User model has fields that have already been implemented on Django's auth user model like firstname, lastname, email, etc...

File: apps/users/models.py [hunk 1]
```diff
@@ -1,3 +1,29 @@
-# from django.db import models
+from django.db import models
+from django.conf import settings
+
+
+# Enums
+class Roles(models.TextChoices):
```
Comment (AvivLiberman) [inline apps/users/models.py:6]: After we merge #45 try using enumchoicefield

**Pull request closed without merging**